python/src/trezorlib/cli/settings.py

```diff
@@ -132,6 +132,20 @@ def homescreen(client, filename):
     return device.apply_settings(client, homescreen=img)
 
 
+@cli.command()
+@click.argument("allow", type=click.Choice(("on", "off")))
```

Comment:
Here we use "on/off" and for passphrase for example we use `trezorctl set passphrase enabled`. We should unify those IMHO. Also "on" might be the default value? Maybe let's create a separate issue for that?

Reply:
#1130
I am not sure what to do here. I chose "on/off" instead of "enabled/disabled" because of a vague feeling that "set unsafe-prompts disabled" sounds like the less secure setting. But that's not very strong argument. Do you think we should change it to `enabled/disabled`?

Reply:
I like on/off. I wouldn't mind having also "trezorctl set passphrase on". Anyway, since we are not sure I suggest to leave this as is and deal with it in #1130.

Reply:
> I chose "on/off" instead of "enabled/disabled" because of a vague feeling that "set unsafe-prompts disabled" sounds like the less secure setting.

I don't think on/off vs. enabled/disabled makes a difference in this respect. If a configuration sounds less secure it's more likely because of the name of the setting.